### DISCUSSION TITLE
small objects detection using sahi or combine it with other models

### DISCUSSION THREAD

**Discussion by merbenf** [roboflow/supervision#1197] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

 i just wanted to ask for "sahi" model wich is used for small objects detection how can i use it in inference with faster r cnn and if there any other models are used for very small (tiny) objects to be used .
thank you all .

### Additional

i already use it with yolov5 and know i wanna test with other models such as faster r cnn to reach the best performance

**Comment by SkalskiP** [comment]
Hi @merbenf 👋🏻 We have a whole docs page dedicated to SAHI with usage examples: https://supervision.roboflow.com/develop/how_to/detect_small_objects/

**Comment by SkalskiP** [comment]
I'll convert this issue into a discussion and put it into the Q&A section.
